.github/workflow/black.yml

```diff
@@ -0,0 +1,10 @@
+name: Lint
```

Comment:
close!  but you can tell it didn't run because the PR doesn't show any of the checks passed/failed like mine did.  

I think you just need to name the directory `workflows` (with an s) and it'll pick them up

Reply:
Is there a good way to rename the directory? All I could manage to do was edit the individual files and then "rename" them with a different directory (workflows). Didn't see a directory rename.

Reply:
Depends on how/where you're doing this - but I think VSCode just lets you edit folder names on the panel.  You'd then have to `git add` the new ones.  i think you can also `git mv <original_location> < new_location>` but don't quote me on that.